Exawind/exawind-benchmarks
# Add mesh files using dvc

This is a draft pull request. DO NOT MERGE!

To test this, do 

```
git clone --single-branch --branch f/dvc git@github.com:gantech/exawind-benchmarks.git
cd exawind-benchmarks
dvc pull
```

If someone can verify that this works, then I can push the mesh files to the s3 bucket on the amazon cloud provided by PNNL through wind data hub.
